DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -959,11 +959,13 @@ private void RefreshPlayerList(object sender, EventArgs e)
         {
             string selectedUserName = lbPlayerList.SelectedItem == null ?
                 string.Empty : lbPlayerList.SelectedItem.Text;
+
             lbPlayerList.Clear();
 
             foreach (ChannelUser user in currentChatChannel.Users)
             {
-                AddUser(user);
+                user.IsFriend = pmWindow.IsFriend(user.IRCUser.Name);
```

Comment:
It'd be better to set this property when the user joins CnCNet (in CnCNetManager), although that'd need splitting the friend-handling functions from the PM window so this is a fine temporary hack.

Reply:
I did see that too, I didn't want to do that in this PR though.